java/runtime/src/main/java/org/ray/runtime/MultiRuntimeHelper.java

```diff
@@ -0,0 +1,19 @@
+package org.ray.runtime;
+
+import org.ray.api.id.UniqueId;
+import org.ray.runtime.task.TaskExecutor;
+import java.util.HashMap;
+
+public class MultiRuntimeHelper {
```

Comment:
MultiWorkerRuntimeHelper? 

This file can be renamed as well.

Reply:
also the name `helper` itself seems a bit more generic than what it actually does.

Reply:
how about just add a static `getTaskExecutorByWorker` in `TaskExecutor`? 
(we can set the mapping in `TaskExecutor`'s constructor.

Reply:
Thanks. I put these methods into `TaskExecutor`

Reply:
Thanks. I put these methods into TaskExecutor.